src/rag/embeddings.py

```
@@ -1,9 +1,141 @@
+                model=model,
+            )
+
+            batch_embeddings = [item.embedding for item in response.data]
+            all_embeddings.extend(batch_embeddings)
+
+        # 원본 인덱스에 맞게 결과 재배열 (빈 텍스트는 0 벡터)
+        embedding_dim = len(all_embeddings[0]) if all_embeddings else int(
+            os.getenv("EMBEDDING_DIMENSION", "1536")
+        )
+        result = [[0.0] * embedding_dim] * len(texts)
+        for idx, embedding in zip(original_indices, all_embeddings):
```

Ruff (0.15.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)